src/utils/upload/fileInspector.js

```diff
@@ -34,11 +34,15 @@ const inspectFile = async (file, technology) => {
     // only extension is checked
     return Verdict.VALID_ZIPPED;
   } if (technology === sampleTech.RHAPSODY) {
+    // TODO: check why we would forbid non-gzipped files
     return inspectRhapsodyFile(file);
-  }
-  if (technology === sampleTech.H5) {
+  } if (technology === sampleTech.H5) {
```

Comment:
Doesn't affect this PR in particular, but I think it would be interesting to look into unifying this `fileInspector` and the `sampleValidators` in some kind of consistent way or at the very least clearly define and comment separation of responsibilities.